server/events/vcs/github_client.go

```diff
@@ -121,23 +122,28 @@ func NewGithubClient(hostname string, credentials GithubCredentials, config Gith
 		return nil, errors.Wrap(err, "error initializing github authentication transport")
 	}
 
+	transportWithRateLimit, err := github_ratelimit.NewRateLimitWaiterClient(transport.Transport, github_ratelimit.WithTotalSleepLimit(time.Minute, nil))
```

Comment:
regarding `WithTotalSleepLimit(time.Minute, nil)` - not sure how long we want to keep retry, and if this should be configurable

Reply:
We are trying to limit as much as possible adding more flags, so maybe we can add this to the repo.Yaml instead.

@chenrui333

Reply:
is something like this acceptable? open to suggestions
``` yaml
...
git:
  github:
     secondary_rate_limit_duration: 60
```

Reply:
yeah, repo config would be a good idea.

Reply:
Can I suggest starting with the hard-coded 1 minute value that you currently have, so that we can get this PR finished. It can always be refined later to make it configurable.